virtual/libudev/libudev-251-r2.ebuild

```diff
@@ -12,14 +12,16 @@ KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv
 IUSE="eudev +sticky-tags systemd"
 REQUIRED_USE="
 	?? ( eudev systemd )
-	eudev? ( !sticky-tags )
 "
 
-# eudev does not provide v251 APIs, see
-# https://github.com/eudev-project/eudev/issues/249
 RDEPEND="
 	!systemd? (
-		eudev? ( sys-fs/eudev[${MULTILIB_USEDEP}] )
+		eudev? (
+			sticky-tags? (
+				>=sys-fs/eudev-3.2.12-r2[${MULTILIB_USEDEP}]
+			)
+			sys-fs/eudev[${MULTILIB_USEDEP}]
```

Comment:
The sticky-tags USE flag was meant as temporary hack.

Please drop the sticky-tags USE flag and depend >=sys-fs/eudev-3.2.12-r2 unconditionally.

Reply:
Sure, I changed it.